addons/ofxEmscripten/src/ofxAppEmscriptenWindow.cpp

```diff
@@ -98,17 +99,107 @@ void ofxAppEmscriptenWindow::display_cb(){
 
 int ofxAppEmscriptenWindow::keydown_cb(int eventType, const EmscriptenKeyboardEvent *keyEvent, void *userData){
 	int key = keyEvent->key[0];
-	if(key==0){
+	std::string id = keyEvent->key;
+	if(key == 0){
 		key = keyEvent->which + 32;
+	}else if(id == "ArrowLeft"){
+		key = 57356;
+	}else if(id == "ArrowUp"){
+		key = 57357;
+	}else if(id == "ArrowRight"){
+		key = 57358;
```

Comment:
Might be good to use the ofKey defines for these? 
eg: `key = OF_KEY_RIGHT`

Reply:
Definitely...

Reply:
@ofTheo i think its better now and can be merged.